scripts/gh-projects/examples/mux-video-integration/phase-1/c1.md

```
@@ -0,0 +1,9 @@
+Add the `@mux/mux-player-astro` package as a runtime dependency.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)